go.mod

```diff
@@ -290,6 +290,9 @@ require (
 )
 
 replace (
+	// TODO MUST BE REMOVED IN THE NEXT FOLLOWING PR (or this PR needs to be split and cosmossdk.io/api tagged)
```

Comment:
Can we split even more, so we don't need that?

Reply:
ill remove this in the next follow up pr. this flow is really annoying. we should figure something out

Reply:
👍, I think that's the best to keep this workflow so we don't forget and end up never tagging our changes

Reply:
+1, I would do 1 PR for proto files only. This way we can split the module's public API and implementation, and the API PR can get merged in <1d.

Can be for future PRs though, if @marbar3778 you want to keep this one.

Reply:
the pr that this was split out of will have the removal. a single pr for api module isn't needed if there is a follow up already imo